src/releasers/java/version.ts

```diff
@@ -14,49 +14,57 @@
 
 import {BumpType} from './bump_type';
 
-const VERSION_REGEX = /(\d+)\.(\d+)\.(\d+)(-\w+)?(-SNAPSHOT)?/;
+const VERSION_REGEX = /(\d+)\.(\d+)\.(\d+)(-.+)?/;
```

Comment:
wondering if it's worth pulling these files into a `utils/` subfolder, maybe it's not worth the extra nesting? i'm on the fence.